packages/components/dialog/src/use-dialog.ts

```diff
@@ -45,13 +44,13 @@ export const useDialog = (
 
   const style = computed<CSSProperties>(() => {
     const style: CSSProperties = {}
-    const varPrefix = `--${namespace.value}-dialog`
+    const varPrefix = `${namespace.value}-dialog`
```

Comment:
I think it's better to use the const assertion directly here.

```ts
const varPrefix = `--${namespace.value}-dialog` as const
```

Reply:
What about using `module enhancements`?

Reply:
<img width="942" alt="image" src="https://user-images.githubusercontent.com/50739490/179892158-1d8ffb80-d526-4f8e-95ec-32592b54d25c.png">

Reply:
> I think it's better to use the const assertion directly here.
> 
> ```ts
> const varPrefix = `--${namespace.value}-dialog` as const
> ```

The const assertion here is to narrow the type `string` to ``` `--${string}` ```, what do you mean?

Reply:
Updated. Thanks for your advice